code/game/objects/items/devices/traitordevices.dm

```diff
@@ -404,3 +404,22 @@
 
 	times_used = max_uses - 1
 	activate_batterer()
+
+/obj/item/handheld_mirror
+	name = "hand mirror"
+	desc = "Style, on the go!"
+	icon = 'icons/obj/device.dmi'
+	icon_state = "hand_mirror"
+	w_class = WEIGHT_CLASS_TINY
+	var/datum/ui_module/appearance_changer/appearance_changer_holder
```

Comment:
this needs to be gc'd properly

Reply:
GC?

Reply:
Garbage collection, essentially making sure things that are deleted get deleted as efficiently as possible as possible